src/api/utils/getTodos.ts

```diff
@@ -0,0 +1,4 @@
+import { API_URL } from '../constants/constants';
+import { getData } from './getData';
+
+export const getTodos = (): Promise<Todo[]> => getData<Todo[]>(`${API_URL}/todos`);
```

Comment:
если ты описала функцию getData, то описывать что отдает текущая не нужно
```suggestion
export const getTodos = () => getData<Todo[]>(`${API_URL}/todos`);
```